pyproject.toml

```diff
@@ -165,13 +169,29 @@ enable_error_code = ["ignore-without-code", "redundant-expr", "truthy-bool"]
 warn_unreachable = true
 disallow_untyped_defs = false
 disallow_incomplete_defs = false
```

Comment:
there may be spurious options in the `mypy` config now (given changed defaults)

Reply:
I believe those are all documented here: https://kotlinisland.github.io/basedmypy/error_code_list.html

Reply:
```toml
warn_unused_configs = true
strict = true
enable_error_code = ["ignore-without-code", "redundant-expr", "truthy-bool"]
warn_unreachable = true
```

these are all enabled by default in basedmypy